app/components/task_list_items/cil_liability_component.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module TaskListItems
+  class CilLiabilityComponent < TaskListItems::BaseComponent
+    def initialize(planning_application:)
+      @planning_application = planning_application
+    end
+
+    private
+
+    attr_reader :planning_application
+
+    delegate(:cil_liable, to: :planning_application)
+
+    def link_text
+      "CIL Liability"
+    end
+
+    def link_path
+      edit_planning_application_cil_liability_path(
+        planning_application
+      )
+    end
+
+    def status
+      if cil_liable.nil?
+        :not_started
+      else
+        "cil_liable_#{cil_liable}".to_sym
+      end
```

Comment:
Since `cil_liable` is a boolean, dynamically creating strings which will need to be GC'd and converting them to symbols doesn't seem the best way to implement this, how about:

```
def status
  case cil_liable
  when TrueClass
    :cil_liable_true
  when FalseClass
    :cil_liable_false
  when NilClass
    :not_started
  else
    raise RuntimeError, "Unexpected value for `cil_liable': #{cil_liable.inspect}"
  end
end
```

Also, would the statuses not be better as `not_started`, `cil_liable`, `not_cil_liable` ?